isis/src/qisis/objs/ControlPointEditWidget/ControlPointEditWidget.cpp

```diff
@@ -562,6 +559,72 @@ namespace Isis {
   }
 
 
+  /**
```

Comment:
having some trouble understanding this header

Reply:
I changed to:

  /**
   * Fill m_projectShapeNames with ALL shapes currently in project. The first 
   * m_numberProjectShapesWithPoint actually contain the location of m_editPoint.
   *
Does that help?